live/src/agama-installer.changes

```diff
@@ -1,3 +1,11 @@
+-------------------------------------------------------------------
+Tue May 13 16:56:56 UTC 2025 - Ladislav Slezák <lslezak@suse.com>
+
+- Do not set the root in /boot/grub2/grub.cfg file, it was wrong
+  and the root is already set in the /boot/grub2/earlyboot.cfg
+  file (bsc#1236873)
+- Replace underscores in the boot menu label by spaces
```

Comment:
would it make sense to replace both? like underscore and also dashes?

Reply:
Dashes actually cannot be used as the Kiwi profile name is then used as a part of the version in OBS and dash is not allowed in RPM version.